Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionCiscoMeraki.yaml

```diff
@@ -180,6 +182,8 @@ ParserQuery: |
             | extend NetworkIcmpType = iff(isnotempty(NetworkIcmpCode), coalesce(NetworkIcmpType_lookup, NetworkIcmpType), "")
             | extend pattern = coalesce(pattern1, pattern2)
             | lookup DvcActionLookup on pattern
+            | extend direction = case(pattern has_any ('0','1'), 'ingress', pattern has_any ('allow','deny'), 'egress', 'unknown')
+            | lookup NetworkDirectionLookup on direction
```

Comment:
@tduarte14 thanks for suggesting these changes. Please add the same changes to the 'vim' (filtering) parser for meraki as well.

https://github.com/Azure/Azure-Sentinel/blob/efc04c5d66c0b4720942bff5273114e2c7051536/Parsers/ASimNetworkSession/Parsers/vimNetworkSessionCiscoMeraki.yaml

Reply:
Changes done in the vim yaml file also